fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PushDownPredicateAggRule.java

```diff
@@ -39,10 +39,12 @@ public List<OptExpression> transform(OptExpression input, OptimizerContext conte
             List<ColumnRefOperator> columns = Utils.extractColumnRef(scalar);
 
             // push down predicate
+            // may be some problems. e.g. select count(*) from tbl having false return empty results.
+            // while select count(*) from (select * from tbl where false) t return one row with result of 0.
```

Comment:
confirm the problem plz？

Reply:
Read this code again. It's not a problem. I have deleted this comment.